mu/locale/pl/LC_MESSAGES/mu.po

```diff
@@ -5,29 +5,26 @@
 msgid ""
 msgstr ""
 "Project-Id-Version: \n"
-"POT-Creation-Date: 2018-09-07 18:34+CEST\n"
-"PO-Revision-Date: 2018-09-11 18:23+0200\n"
+"POT-Creation-Date: 2019-04-30 19:11+CEST\n"
```

Comment:
This looks like an unrelated language patch, could you submit it in a different PR?